python/dgl/graphbolt/dataloader.py

```diff
@@ -122,7 +187,35 @@ def __init__(self, datapipe, num_workers=0, persistent_workers=True):
             persistent_workers=persistent_workers,
         )
 
-        # (3) Cut datapipe at CopyTo and wrap with prefetcher. This enables the
+        # (3) Overlap UVA feature fetching by buffering and using an alternative
+        # stream.
+        if (
+            overlap_feature_fetch
+            and num_workers == 0
+            and torch.cuda.is_available()
+        ):
+            torch.ops.graphbolt.set_max_uva_threads(max_uva_threads)
```

Comment:
if torch cuda is available, can't we assume that GRAPHBOLT_USE_CUDA is turned on?

Reply:
I can compile the newly added cc file even if GRAPHBOLT_USE_CUDA is off if you want, I just need to move the addition of the source file outside of the USE_CUDA guards in both cmake and the python_binding.cc

Reply:
I am actually suggestion remove the flag since I don't really understand the benefit. We probably only need to make sure that for pure CPU build, we can ignore the cuda code.